src/openMVG/multiview/solver_resection_p4pf.hpp

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
ah, I thought he already made those changes.
@michalpolic can you also add those changes? thx